docs/build/quick_start.md

```diff
@@ -0,0 +1,236 @@
+
+```sh
+# Use pipx to install ninja if not already installed
+pipx install ninja
+pipx ensurepath
+# relaunch your shell after installation
+exec $SHELL
+```
+
+## Start develop C++ problem solver of ostis-metasystem with Conan
+
+### Install Conan
+
+Install Conan, to build C++ problem solver dependencies with Conan-provided dependencies:
+
+```sh
+# Use pipx to install conan if not already installed
+pipx install conan
+pipx ensurepath
+# relaunch your shell after installation
+exec $SHELL
+```
+
+### Use C++ problem solver in Debug
+
+#### Install dependencies with Conan
+
+C++ problem solver of ostis-metasystem is an extension to sc-machine, so sc-machine is main dependency for C++ problem solver. To install it, run in the root of the project:
+
+```sh
+conan install . -s build_type=Debug --build=missing
+```
+
+#### Build C++ problem solver in Debug
+
+To build C++ problem solver in debug mode using Conan-provided dependencies, run:
+
+```sh
+# debug build type
+cmake --preset debug-conan
+cmake --build --preset debug
+```
+
+!!! Note
+    By default, configure preset `debug` enables building C++ problem solver tests.
+
+#### Run C++ problem solver tests in Debug
+
+After that, you can go to `build/Debug` and run tests via `ctest`:
+
+```sh
+cd build/Debug
+ctest -V
+```
+
+You can also check code formatting. To learn more, go to the [CMake flags](cmake_flags.md) page.
+
+### Use C++ problem solver in Release
+
+#### Install dependencies with Conan
+
+To install it, run in the root of the project:
+
+```sh
+conan install . -s build_type=Release --build=missing
+```
+
+#### Build C++ problem solver in Release
+
+To build C++ problem solver in release mode using Conan-provided dependencies, run:
+
+```sh
+# release build type without tests
+cmake --preset release-conan
+cmake --build --preset release
+```
+
+To build C++ problem solver with tests in release mode using Conan-provided dependencies, run:
+
+```sh
+# release build type with tests
+cmake --preset release-with-tests-conan
+cmake --build --preset release
+```
+
+#### Run C++ problem solver tests in Release
+
+After that, you can run tests:
+
+```sh
+cd build/Release
+ctest -V
+```
+
+You can also check code formatting. To learn more, go to the [CMake flags](cmake_flags.md) page.
+
+## Start develop Python problem solver of ostis-metasystem
+
+### Install dependencies with pip3
+
+To install dependencies, run:
+
+```sh
+python3 -m venv platform-dependent-components/problem-solver/py/.venv
+source platform-dependent-components/problem-solver/py/.venv/bin/activate
+pip3 install -r platform-dependent-components/problem-solver/py/requirements.txt
+```
+
+### Run Python problem solver tests
+
+Running Python problem solver requires C++ problem solver to be running. Run C++ problem solver:
+
+```sh
+./build/Release/bin/sc-machine -s kb.bin \
+    -e "path/to/ostis-metasystem/lib/extensions;path/to/sc-machine/lib/extensions"
```

Comment:
no sc-machine in bin folder
![image](https://github.com/user-attachments/assets/e6351fe4-1fbd-409e-a631-f451063b57a3)